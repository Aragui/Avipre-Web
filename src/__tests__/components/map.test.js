import {render} from '@testing-library/react'

import Map from '../../components/map'

describe('<Map />', () => { 
    const {container} = render(<Map />)

    test('Render Map', () => { 
        expect(container).toMatchSnapshot()
     })
 })