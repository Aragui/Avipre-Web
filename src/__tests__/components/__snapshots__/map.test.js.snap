// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> Render Map 1`] = `
<div>
  <div
    id="ubicacion"
  >
    <iframe
      height="450"
      loading="lazy"
      referrerpolicy="no-referrer-when-downgrade"
      src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3724.465253441941!2d-89.61212008522136!3d21.014062186006306!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8f5676b96b60f7a3%3A0x4532a4c5af95b0cc!2s28St%20Coworking%20Center!5e0!3m2!1sen!2sus!4v1653072447560!5m2!1sen!2sus"
      style="border: 0px;"
      width="100%"
    />
  </div>
</div>
`;
